.rubocop_local.yml

```diff
@@ -3,3 +3,6 @@ GlobalVars:
     # Loggers
     - $log
     - $azure_stack_log
+
+Naming/ClassAndModuleCamelCase:
+  Enabled: false
```

Comment:
Turning this off for entire codebase feels a bit strong, since we only have issues with the two classes `V2018_03_01`and `V2017_03_09`  I think we should just add

```
# rubocop:disable Naming/ClassAndModuleCamelCase
```

comment to the problematic lines (where we define classes), see https://rubocop.readthedocs.io/en/stable/configuration/#disabling-cops-within-source-code